src/policies/conditions.ts

```diff
@@ -459,9 +444,9 @@ export class ConditionContext {
   }
 
   public toJson = async (): Promise<string> => {
-    const userAddressParam = this.contextParameters.find(
```

Comment:
I think this was just checking to see if `:userAddress` was included in any of the conditions. If I am wrong about this please let me know

Reply:
Yes, that's correct.